brightStars_DM_sprint_Feb2020/BeyondRadius.ipynb

```diff
@@ -0,0 +1,422 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "for tract, dens in zip(tractList, densities):\n",
+    "    # average detection densities around stars in this magnitude bin:\n",
+    "    magDen = np.mean(dens,axis=0)\n",
+    "    magDenStd = np.std(dens,axis=0)\n",
+    "    plt.errorbar(distanceToEdge, magDen, yerr=magDenStd, capsize=2)\n",
+    "    plt.plot(distanceToEdge, np.min(dens,axis=0), c='red', ls='--')\n",
+    "    plt.plot(distanceToEdge, np.max(dens,axis=0), c='red', ls='--')\n",
```

Comment:
A legend would be nice to indicate what the lines are (and the distinction between the red-dashed lines and the error bars on the blue lines).